src/objects/p5.Graphics3D.js

```diff
@@ -2,6 +2,7 @@ define(function(require) {
 
   var p5 = require('core');
   var shaders = require('shaders');
+  require('p5.Shader');
```

Comment:
If #720 lands before this does, this will need to be changed to `'src/objects/p5.Shader'`. If this lands before #720 does, that PR will need to be rebased onto this one to account for the change.